hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -157,6 +157,10 @@ private Constants() {
       "fs.s3a.connection.ssl.enabled";
   public static final boolean DEFAULT_SECURE_CONNECTIONS = true;
 
+  // allow access to requester pay buckets
```

Comment:
nit: make a javadoc, with a reference to {@value} in it. Good for IDE popup hints